table_schema_cache.go

```diff
@@ -24,6 +24,11 @@ type TableIdentifier struct {
 	TableName  string
 }
 
+type MinMaxKeys struct {
```

Comment:
This looks almost the same as `PaginationKey` - could we collapse these 2 structs?

Reply:
Agree, fixed.